### DISCUSSION TITLE
Weekly granularity model and seasontype field in ets_f function

### DISCUSSION THREAD

**Discussion by Miky14880** [Nixtla/statsforecast#202] [Q&A]
Ciao
I was using the ETS module and I tried to use it for a weekly granularity forecast by putting season_length=52 without success.
Checking the code looks like at the moment weekly granularity is not managed as season_length cannot be greater than 24, is my understanding correct?
Always reviewing the code I noticed that seasontype filed can have the following values N,D,M,A,Z. I think i have understood the meaning of the first 3 chars, but it's not clear to me the meaning of Z,  any idea?

Thanks in advance for the support and hats off for the great package!
Regards
Michela

**Comment by kdgutier** [comment]
Hi @Miky14880,

Autorregresive dependencies of such length are challenging for sequential models like ARIMA and ETS.
Have you tried to encode the days of interest using dummies and add them as regressors?

**Comment by AzulGarza** [comment]
hey @Miky14880! The latest version of statsforecast supports `season_length=52` for the `AutoETS` model. :)
